README.md

```diff
@@ -101,6 +101,12 @@ The gem [includes integration tests](lib/govuk_publishing_components/minitest/co
 
 Automated accessibility tests use [aXe](https://github.com/dequelabs/axe-core). Using our `AccessibilityTest` wrapper the gem runs the aXe suite against each example and throws JavaScript errors for any violations. These JavaScript errors can be used to fail a build in CI.
 
+### Exclude accessibility rules
+
+Sometimes you will have a component that will through an error due to it being in isolation, for example radio buttons not being in a fieldset.
```

Comment:
Should it be "throw" instead of "through"?

Reply:
brain fart cheers :)